src/pages/MonthlyCuration/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useState, useCallback } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import { monthlyCuration } from '@/api/monthly-curation';
+import DummyImage from '@/assets/images/dummy-image.png';
+import Header from '@/components/Header';
+
+import styles from './style.module.scss';
+
+const MonthlyCuration = () => {
+  const [monthlyCurationProducts, setMonthlyCurationProducts] = useState([]);
+  const [title, setTitle] = useState('5월의 선물 큐레이션');
+  //5월일 때 month+1 제거. 햔재 4월
+  const [date, setDate] = useState<number[]>([new Date().getFullYear(), new Date().getMonth() + 1]);
+
+  useEffect(() => {
+    getMonthlyCuration();
+  }, [date]);
+
+  useEffect(() => {
+    console.log(monthlyCurationProducts);
+  }, [monthlyCurationProducts]);
+
+  const handleDateValues = useCallback(
+    (value: string) => {
```

Comment:
다음과 같이 명시해주시면 더 좋을 것 같습니다.

```suggestion
    (value: 'prev' | 'next') => {
```